inloop/common/templatetags/markdown.py

```diff
@@ -5,11 +5,23 @@
 from markdown import Markdown
 from markdown.extensions.codehilite import CodeHiliteExtension
 
+"""Markdown Package.
```

Comment:
If this is supposed to be a module-level docstring, it should be moved to the beginning of the file. Docstrings must appear at the beginning of a file, or immediately follow a `class` or a `def` statement (this is why the Codacy check fails).

Otherwise, it is a very good idea to include this information! :+1:

Reply:
I moved the docstring to the beginning of the file and added a new test case. Please review my commit 👍